paasta_tools/cli/cmds/logs.py

```diff
@@ -559,6 +610,86 @@ def cluster_to_scribe_env(self, cluster):
             return env
 
 
+def generate_start_end_time(from_string="30m", to_string=None):
+    """
+    Parses the --from and --to arguments to create python datetime objects representing
+    the start and end times for log retrieval
+    """
+    if to_string is None:
+        end_time = datetime.datetime.utcnow()
+    else:
+        # Try parsing as a a natural time duration first, if that fails move on to
+        # parsing as an ISO-8601 timestamp
+        to_duration = timeparse(to_string)
+
+        if to_duration is not None:
+            end_time = datetime.datetime.utcnow() - datetime.timedelta(seconds=to_duration)
+        else:
+            end_time = isodate.parse_datetime(to_string)
+
+    from_duration = timeparse(from_string)
```

Comment:
It is, it's because the default argument for to_string is None, an alternative way to handle it would be to make the default to_string be something like "0s" but that just seems hacky.